nebula-logger/main/logger/classes/LogEntryBuilder.cls

```diff
@@ -114,6 +114,11 @@ public without sharing virtual class LogEntryBuilder {
     public LogEntryBuilder setRecordId(SObject record) {
         if(!this.shouldSave) return this;
 
+        if(LoggerSettings__c.getInstance().StoreRelatedRecordJson__c) {
```

Comment:
it seems like even in Logger.cls there are some inconsistencies between the use of the private static final `SETTINGS` variable and simply calling `getInstance` - your thoughts?

Reply:
Yeah, I'll update this PR to eliminate `Logger.getSettings()` and `Logger.SETTINGS` - I think it's easy enough to just call `LoggerSettings__c.getInstance()` whenever needed

Reply:
Sounds good

Reply:
Done - `Logger.getSettings()` and `Logger.SETTINGS` have been replaced with `LoggerSettings__c.getInstance()`

Reply:
Nice - let's get this merged!

Reply:
Will do! After we get your quiddity PR merged next week, I'll circle back to adding extra unit tests for this new functionality & other existing functionality. Since I was bad months ago & didn't originally add tests in `LogEntryBuilder_Tests`, I think it'll cause merge conflicts (either for you or me) if I try to add test methods right now, since you also started filling out `LogEntryBuilder_Tests` in your PR #28. So I'll circle back to unit tests sometime in the next few weeks.

Reply:
Yeah it's always a pain in situations like this. If you _do_ want to add tests in `LogEntryBuilder_Tests` at least related to this functionality as part of this PR, you're certainly welcome to - I'm happy to deal with the merge conflicts. otherwise we can put something in the backlog and I'll pick it up.